ocaml/xapi/xapi_database_backup.ml

```diff
@@ -191,9 +191,16 @@ let handle_notifications __context (d:delta) =
   List.iter ((fun __context (d:del_tables) -> send_notification __context "del" {tbl=d.table; row=d.key}) __context) d.deletes;
   List.iter (send_notification __context "mod") (info_extract d.tables)
 
+let write_db_to_disk =
+  debug "Writing slave db to slave disk";
+  Db_cache_impl.sync (Db_conn_store.read_db_connections ()) !Xapi_slave_db.slave_db
+
 let endless_loop ~__context () =
   let uri = Constants.database_backup_uri in
+  let delay_f = 300.0 in
```

Comment:
We should move this to Xapi_globs and make it configurable from the config file (like the other one you did).